modules/ROOT/pages/afm-modify-monitor.adoc

```diff
@@ -0,0 +1,42 @@
+= Modify a Monitor
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+You can modify monitors via the Functional Monitoring UI, which includes the options to edit the monitor as text fields or to edit the monitor as BAT source. Additionally you can download the monitor as BAT source, edit the monitor in your text editor of choice, and then upload the monitor.
+
+This procedure describes modifying monitors via the Functional Monitoring UI.
+
+== Procedure
+
+There are two methods to modify a monitor.
+
+To modify a monitor from the homepage:
+
+. Hover over the monitor to modify. 
+. Click *Edit* (image:afm-ui-edit-button.png[width=1.5%,height=1.5%]).
+
+To modify a monitor from its detail page:
+
+. Select the monitor that you want to modify.
+. Click the monitor.
```

Comment:
What is the difference between these two steps?
I'm assuming the first is from a menu or list?
And the second is...?